sycl/source/handler.cpp

```diff
@@ -6,6 +6,7 @@
 //
 //===----------------------------------------------------------------------===//
 
+#include "sycl/detail/native_cpu.hpp"
```

Comment:
Why this include is needed here? Shouldn't it be somewhere else considering that there are no other changes in the file?

Reply:
This include is no longer needed and is now removed. Thanks!